### ISSUE TITLE
Kasa devices leaking through via skills (1/2 bug report, 1/2 feature request)

### ISSUE THREAD

**Issue by Abba-** [alandtse/alexa_media_player#2637]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/alandtse/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**

According to the documentation “It will NOT add smart devices connected to Alexa via Skills.”
However, *some* Kasa devices do show up and are controllable.  (Personally this is a **feature!!**, not a bug).

I have ~7 devices (outdoor dual plug, dual outlet, individual plugs).  I had 1 individual plug leak through.
After enabling advanced debugging and parsing the list of "Found unsupported device" the first difference I noticed was I had set the type of plug in Alexa (i.e. customerDefinedDeviceType) for the working switch.  After manually picking that, the remaining smart plugs populated!

Further comparing to the dual devices that didnt appear, they return an additional key that isnt in the smart plug's output:
```
'softwareVersion': '1.0.1 Build 201109 Rel.150108', 'parent.deviceId': '8006E83A21DA6FC8F54924152D8F63C920XXXXXX'}}, 'firmwareVersion'......
```

Presumably that key is what's preventing this from working.
_To be clear, its a bug report in the sense that it goes against the documentation, but this would be an amazing feature to have, even if not fully supported!_

**To Reproduce**
Link some Kasa smart plugs and/or other devices to Alexa.
Set device type in Alexa
Smart plugs show in HA, Dual devices don't.  

**Expected behavior**

According to the documentation, Kasa devices shouldn't show up.
_However, I would humbly prefer this behavior be extended! Not eradicated!_

**Screenshots**

**System details**

- Home Assistant version:
- alexa_media version (from `const.py` or HA startup log):  4.13.5
- alexapy version (from `pip show alexapy` in homeasssistant container or HA startup log):    1.29.3
- Is Amazon [2FA/2SV enabled](https://github.com/alandtse/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account)  (y/n):  y
- Amazon Domain: .com

**Debug Logs (alexa_media & alexapy)**
Please provide [logs](https://github.com/alandtse/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component).

Ill provide a specific section as requested, too much censoring needing to paste everything.

**Additional context**

**Comment by danielbrunt57** [comment]
Do you have the [TP-Link Smart Home](https://www.home-assistant.io/integrations/tplink/) integration installed?
It should add the devices locally into HA for you.

**Comment by Abba-** [comment]
Yes I used it for other installs. The use case here is for a few remote plugs that are on a different network/location and it's impractical to link the networks.

**Comment by danielbrunt57** [comment]
Got it.  Have to see if I can borrow the dual plug outdoor from my gf that I installed for her for patio fountain but serves double duty for Christmas lighting. Could be a couple weeks before I can test...

**Comment by Abba-** [comment]
No rush, Thanks (I waited long enough lol)

Definitely don't risk ruining Christmas automations for this 🙃

**Comment by Abba-** [comment]
Keeping active
